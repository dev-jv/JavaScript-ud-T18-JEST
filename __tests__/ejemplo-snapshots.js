const cliente = {
    nombre: 'Dean Winchester',
    balance: 500,
    tipo: 'Premium'
};

describe('Testing al Cliente', () => {

    test('Es Dean Winchester', () => {
        expect(cliente).toMatchSnapshot();
    });

}) 