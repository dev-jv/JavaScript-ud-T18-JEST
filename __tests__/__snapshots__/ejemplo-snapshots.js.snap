// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al Cliente Es Dean Winchester 1`] = `
Object {
  "balance": 500,
  "nombre": "Dean Winchester",
  "tipo": "Premium",
}
`;
